make-release.sh

```diff
@@ -166,11 +166,12 @@ fetchAndCheckout ()
   git fetch origin "${bBRANCH}:${bBRANCH}"; git checkout "${bBRANCH}"
 }
 
-# unlike in che-plugin-registry, here we just need to update the VERSION file
-updateVersionFile () {
+updateVersion () {
   thisVERSION="$1"
   # update VERSION file with VERSION or NEWVERSION
   echo "${thisVERSION}" > VERSION
+  # ensure version of devworkspace-generator is updated
+  sed -i -r -e "s/(\"version\": )(\".*\")/\1\"${thisVERSION}\"/" tools/devworkspace-generator/package.json
```

Comment:
you could probably make this more legible by using single quotes around the regex, so you don't have to `\"` all the quotes in the regex.

Reply:
done using jq